qaframework-bdd-tests/src/test/resources/features/refapp-2.x/015-reports/reports.feature

```diff
@@ -15,3 +15,4 @@ Feature: Reports Management
     And user enters end date
     And user clicks on request report button
     #Then user clicks on view report link
+    
```

Comment:
Any reason for adding white spaces in the last empty line?

Reply:
Resolved